<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Payment</class>
 <widget class="QWidget" name="Payment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Платеж</string>
  </property>
  <widget class="QPushButton" name="pushButtonSavePay">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Сумма платежа:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPayment">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
